cli/defenseclaw/commands/cmd_setup_redaction.py

```
@@ -0,0 +1,1473 @@
+    raw = click.prompt("Buckets (number, comma list, or all)").strip().lower()
+    selected = list(BUCKETS) if raw == "all" else []
+    if not selected:
+        for token in raw.split(","):
+            normalized = token.strip()
+            if normalized in BUCKETS:
+                selected.append(normalized)
+                continue
+            try:
+                selected.append(BUCKETS[int(normalized) - 1])
+            except (ValueError, IndexError):
+                raise click.ClickException(f"invalid bucket selection {token!r}") from None
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)